test/uk/gov/hmrc/gform/submission/RoboticsXMLGeneratorSpec.scala

```diff
@@ -128,12 +128,21 @@ case object RoboticsXMLGeneratorSpec extends Spec {
     val expected: Elem =
       <gform id={formId.value} dms-id={dmsId} submission-reference={submissionRef.value}>{
         removeWhitespace(expectedFields)
-      }<dateSubmitted>02/01/2019</dateSubmitted></gform>
+      }<dateSubmitted>02/01/2019</dateSubmitted><datetimeSubmitted>2019-01-02T00:00:00Z</datetimeSubmitted></gform>
 
     val dateSubmitted = Instant.from(LocalDate.of(2019, 1, 2).atStartOfDay(ZoneId.of("Europe/London")))
 
+    val dateSubmittedBST = Instant.from(LocalDate.of(2019, 6, 1).atStartOfDay(ZoneId.of("Europe/London")))
+
+    val expectedBST: Elem =
+      <gform id={formId.value} dms-id={dmsId} submission-reference={submissionRef.value}>{
+        removeWhitespace(expectedFields)
+      }<dateSubmitted>01/06/2019</dateSubmitted><datetimeSubmitted>2019-05-31T23:00:00Z</datetimeSubmitted></gform>
```

Comment:
I don't think this look good for casual reader.

```
01/06/2019
2019-05-31T23:00:00Z
```
One day is in May another day is in June. I think we should display `datetimeSubmitted` with zone information ie `+01:00`

Reply:
I replaced `ISO_INSTANT` to `ISO_DATE_TIME` that does have zone information